Permanent/Constants/Strings.swift

```diff
@@ -117,5 +117,6 @@ extension String {
     static var shareActionMessage: String { return "ShareActionMessage".localized() }
     static var addMembers: String { return "\(add) \(members)" }
     static var add: String { return "Add".localized() }
+    static var logOut: String { return "LogOut".localized() }
```

Comment:
Here we should use Log out as 2 separate words since it's an action.

https://whitelightconcepts.com/login-vs-log-in-logout-vs-log-out/

Reply:
The actual text is "Log Out".

Reply:
This is only the `var` name.